Persephone/Models/QueueItem.swift

```diff
@@ -6,10 +6,16 @@
 //  Copyright © 2019 Dan Barber. All rights reserved.
 //
 
-import Foundation
+import AppKit
 
-struct QueueItem: Equatable {
+struct QueueItem: Hashable {
   var song: Song
   var queuePos: Int
   var isPlaying: Bool
+
+  func hash(into hasher: inout Hasher) {
```

Comment:
Now that `Song` is `Hashable`, I think that Swift should be able to synthesise this for you?